src/adapter/src/catalog/builtin.rs

```diff
@@ -4349,6 +4376,15 @@ ON mz_internal.mz_cluster_replica_metrics (replica_id)",
     is_retained_metrics_object: true,
 };
 
+pub const MZ_OBJECT_LIFETIMES_IND: BuiltinIndex = BuiltinIndex {
```

Comment:
I don't love the name, but I can't think of a better one 😅

Reply:
Yeah I wasn't sure how to describe create and drop other than using the term "life" 🤔